beamer/tests/conftest.py

```diff
@@ -219,22 +219,25 @@ def config(request_manager, fill_manager, token, token_list):
     account = eth_account.Account.from_key(_LOCAL_ACCOUNT.private_key)
     token.mint(account.address, 300)
     url = URL(ape.config.provider.uri)
-    rpc_urls = {"l2a": URL(url), "l2b": URL(url)}
+    chains = {}
+    for chain_name in ("l2a", "l2b"):
+        chains[chain_name] = ChainConfig(
+            rpc_url=url,
+            min_source_balance=0,
+            confirmation_blocks=0,
+            poll_period=0.5,
+        )
+
     config = Config(
-        rpc_urls=rpc_urls,
         deployment_info=deployment_info,
-        confirmation_blocks={"l2a": 0, "l2b": 0},
         token_checker=TokenChecker(token_list),
         account=account,
         fill_wait_time=0,
         unsafe_fill_time=600,
-        min_source_balance=0,
-        min_source_balance_per_chain={},
         prometheus_metrics_port=None,
-        base_chain_rpc_url=URL(url),
+        base_chain_rpc_url=url,
         log_level="debug",
-        poll_period=0.5,
-        poll_period_per_chain={},
+        chains=chains,
     )
     beamer.agent.metrics.init(config=config, source_rpc_url=url, target_rpc_url=url)
     return config
```

Comment:
I think we should move this agent-related function to under `beamer/tests/agent` i.e. in the agent-specific `conftest.py` there.

Reply:
moved all fixtures related with agent to its own conftest